src/Tribe/Repositories/Event.php

```diff
@@ -1603,9 +1612,11 @@ protected function order_by_date( $use_utc, $after = false ) {
 			true
 		);
 
-		$order = Arr::get_in_any( [ $this->query_args, $this->default_args ], 'order', 'ASC' );
-		$this->filter_query->orderby( [ $meta_alias => $order ], $filter_id, true, $after );
-		$this->filter_query->fields( "MIN( {$postmeta_table}.meta_value ) AS {$meta_alias}", $filter_id, true );
+		$order = $order === null
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 17e290b8d1398bed3fbb2940dcb2278b</sub>